README.md

```diff
@@ -1 +1,112 @@
-# school-library
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 School library <a name="about-project"></a>
+
+**School library** management system is a comprehensive software project designed to streamline the management of students, teachers, and administrative tasks within an educational institution. This multi-phase project consists of seven key components, each aimed at enhancing functionality, maintainability, and usability.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack
+
+This project is built using the following technologies:
+
+<details>
+<summary>Programming Languages</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+<details>
+  <ul>
+    <li>Creation and management of student and teacher profiles.</li>
+    <li>Authentication and authorization to restrict access to authorized users.</li>
+  </ul>
+</details>
+
+
+<!-- GETTING STARTED -->
+
+## Getting Started <a name="getting-started"></a>
+
+To get started with the School library, follow the steps below:
+
+### Prerequisites
+
+Before running this project, make sure you have the following prerequisites installed:
+
+- Ruby
+
+### Installation <a name="installation"></a>
+
+1. Clone this repository to your desired folder:
+
+   ```sh
+   git clone https://github.com/Ochiengsteven/school-library.git
+
+2. Navigate to the project directory:
+
+    ```sh
+    cd school_library
+3. Run the program:
+    ```sh
+    ruby school_library.rb
```

Comment:
- [ ] Great job implementing your decorator, everything works as it's suppose to :clap: However, running `ruby school_library.rb` doesn't start your app as you indicated in your README file because there is not a file named `school_library.rb` in your project. Kindly add the right command to run your application (`ruby test.rb`). That will make your README file look more professional.